src/utils/getVisibleTodos.ts

```diff
@@ -0,0 +1,32 @@
+import { SelectValue } from '../types/SelectValue';
+import { Todo } from '../types/Todo';
+
+export const getVisibleTodos = (
+  todos: Todo[],
+  select: string,
+  query: string,
+): Todo[] => {
+  let filteredTodos = todos.filter(todo => {
+    switch (select) {
+      case SelectValue.ACTIVE:
+        return !todo.completed;
+
+      case SelectValue.COMPLETED:
+        return todo.completed;
+
+      case SelectValue.ALL:
+        return true;
+
+      default:
```

Comment:
if you will set value to select from enum, you'll be sure you won't get unexpected type, and would be sense to move `All` to default case